### PR TITLE
Implement client suggested improvments

### PR DESCRIPTION
**Pull request by marcelvanworkum**
Added the following features:
    - Distance between current location and rocket's location
    - Compass bearing between current location and rocket's location
    - Add a QRCode image to the GUI which allows the Google map location to be opened easily on a phone. This will be super useful when people are in field and want to try and locate the rocket, as using their phone they can see their location in real time.

### CODE REVIEW

File: software/GUI/map_tab.py [hunk 1]
```diff
@@ -81,6 +98,14 @@ def set_map_image(self):
 
         self.map_label.setPixmap(map_image)
 
+    def set_qr_code_image(self):
+        """
+        Checks t see if the qr code image exists, and if it does, sets the qr code image label to that.
```
Comment (jamiesanson) [inline software/GUI/map_tab.py]: "Checks to"

File: software/util/qr_code_generator.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import pyqrcode
+import png
+
+
```
Comment (jamiesanson) [inline software/util/qr_code_generator.py:4]: Doc this

**Review by jamiesanson — APPROVED** [review]
Yeah good thanks

File: software/util/qr_code_generator.py (hunk 2 above)
Comment (marcelvanworkum) [reply software/util/qr_code_generator.py:4]: Changed in #2ae18f7e4b3e535afc67c2bad1946bdd18b2dd91, not sure why it doesn't show up. Must be the new github review thing.

**Pull request merged**